internal/backends/winit/event_loop.rs

```diff
@@ -418,6 +410,8 @@ impl winit::application::ApplicationHandler<SlintEvent> for EventLoopState {
     }
 
     fn about_to_wait(&mut self, event_loop: &ActiveEventLoop) {
+        self.loop_error = self.shared_backend_data.create_inactive_windows(event_loop).err();
```

Comment:
is there the risk to override an existing error with None?

Reply:
Yes indeed. Oops, I'll fix that , as well as not emptying the window list if one produces an error.